templates/cli/lib/parser.js.twig

```diff
@@ -110,13 +114,45 @@ const drawJSON = (data) => {
 }
 
 const parseError = (err) => {
-    if(cliConfig.verbose) {
-        console.error(err);
-    }
+    if (cliConfig.report) {
+        (async () => {
+            let appwriteVersion = 'unknown';
+            const isCloud = globalConfig.getEndpoint().includes('cloud.appwrite.io') ? 'Yes' : 'No';
+
+            try {
+                const res = await fetch(`${globalConfig.getEndpoint()}/health/version`);
+                const json = await res.json();
+                appwriteVersion = json.version;
+            } catch {
+            }
+
+            const version = '{{ sdk.version }}';
+            const stepsToReproduce = `Running \`appwrite ${cliConfig.reportData.data.args}\``;
+            const yourEnvironment = `CLI version: ${version}\nOperation System: ${os.type()}\nAppwrite version: ${appwriteVersion}\nIs Cloud: ${isCloud}`;
+
+            const stack = '```\n' + err.stack + '\n```';
+
+            const githubIssueUrl = new URL('https://github.com/appwrite/appwrite/issues/new');
+            githubIssueUrl.searchParams.append('labels', 'bug');
+            githubIssueUrl.searchParams.append('template', 'bug.yaml');
+            githubIssueUrl.searchParams.append('title', `🐛 Bug Report: ${err.message}`);
+            githubIssueUrl.searchParams.append('actual-behavior', `CLI Error:\n${stack}`);
+            githubIssueUrl.searchParams.append('steps-to-reproduce', stepsToReproduce);
+            githubIssueUrl.searchParams.append('environment', yourEnvironment);
 
-    error(err.message);
+            log(`To report this error you can:\n - Create a support ticket in our Discord server https://appwrite.io/discord \n - Create an issue in our Github\n   ${githubIssueUrl.href}\n`);
```

Comment:
lets also print the stack trace so they can share it with us on discord. Currently the stack trace is encoded in the github url